commit 701cf5b46b7b5fb06d4ab5c7ea8d4e22b4d0cf10 (refs/changes/02/4702/2)
Author: Dan McDonald <danmcd@joyent.com>
Date:   2018-08-21T13:51:07-04:00 (1 year, 2 months ago)
    
    OS-7141 Overlay device-creation weirdness contributes to varpd boot failures

diff --git a/overlay/generic/etc/name_to_major b/overlay/generic/etc/name_to_major
index cfab8bff..26eaeddb 100644
--- a/overlay/generic/etc/name_to_major
+++ b/overlay/generic/etc/name_to_major
@@ -193,6 +193,7 @@ ixgbe 192
 bpf 193
 lsimega 194
 vioif 195
+overlay 196
 mpt_sas 198
 mr_sas 199
 mega_sas 200
